open_city_profile/settings.py

```diff
@@ -213,3 +214,20 @@
                 "Please create a %s file with random characters to generate your secret key!"
                 % secret_file
             )
+
+# Youth profile settings
+
+GENDERS = (
+    ("male", "Male"),
+    ("female", "Female"),
+    ("other", "Other"),
+    ("dont_know", "Don't know"),
+    ("rather_not_say", "Rather not say"),
+)
+
+ILLNESSES = (
+    ("diabetes", "Diabetes"),
+    ("epilepsy", "Epilepsy"),
+    ("heart_disease", "Serious heart or circulatory disease"),
+    ("serious_allergy", "Serious allergy"),
```

Comment:
is this list confirmed by someone from the city side? should it be `serious allergy` specifically? not any allergy?

Reply:
resolved IRL

Reply:
Unfortunately, there hasn't been anyone from the youth services available to confirm these, so I've made do with the documents prepared in their presence: [meeting notes](https://gitlab.com/City-of-Helsinki/dokumentaatio/ict-kehitys/wikis/meetingnotes/2019-04-08-nuorisoasiakkuus) and [UI flow wireframes](https://api.media.atlassian.com/file/0574012d-bb60-4301-8a80-253dee209b83/binary?client=02fbb8b3-7114-499e-bebc-cc4a1e0c1556&dl=true&token=eyJhbGciOiJIUzI1NiJ9.eyJpc3MiOiIwMmZiYjhiMy03MTE0LTQ5OWUtYmViYy1jYzRhMWUwYzE1NTYiLCJhY2Nlc3MiOnsidXJuOmZpbGVzdG9yZTpmaWxlOmE4YmQ0ZjljLTc4MmEtNGY2My05NWE2LTBhMDRlNzhlYmIxZSI6WyJyZWFkIl0sInVybjpmaWxlc3RvcmU6ZmlsZTo2OTc0YjZmNy04ZDdkLTRmMzYtOWE0Yy0zNjM2YTg5ZjAzZjYiOlsicmVhZCJdLCJ1cm46ZmlsZXN0b3JlOmZpbGU6MDU3NDAxMmQtYmI2MC00MzAxLThhODAtMjUzZGVlMjA5YjgzIjpbInJlYWQiXX0sImV4cCI6MTU2NTY4MTA4NCwibmJmIjoxNTY1NjgwMTI0fQ.4NNk6guC72qoIDBvNwsw8HWjwtpw4-9vuoVIppWJIv8).

Reply:
But yeah, these come from "spec" documents...

Reply:
probably should make labels translatable BTW

Reply:
I've now asked about this from the contact person I know, but they might take a while to answer. I suggest we leave it like t his for the time being and modify upon further comments.

Reply:
ok 👍